lib/solarwinds_apm/otel_native_config.rb

```diff
@@ -20,6 +20,8 @@ module OTelNativeConfig
     @@config_map       = {}
     @@agent_enabled    = false
 
+    RESOURCE_ATTRIBUTE = 'RESOURCE_ATTRIBUTE'
+
```

Comment:
Was this meant to go from plural to singular ?

Reply:
I was just creating some name for in-code configuration, but I can change to plural (e.g. `RESOURCE_ATTRIBUTES = 'RESOURCE_ATTRIBUTES'`)